rest-service/manager_rest/plugins_update/manager.py

```diff
@@ -209,36 +219,42 @@ def get_reevaluated_plan(self, blueprint, resolver_parameters):
         return temp_plan
 
 
+def _plugins_in_a_plan(plan):
+    for executor in [DEPLOYMENT_PLUGINS_TO_INSTALL,
```

Comment:
plz choose a different name, `executor` already means "host_agent" or "central_deployment_agent", this name just makes it confusing

Reply:
sure (not that I invented that `executor` ;) )